code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -304,19 +304,19 @@
 		if(op != "")
 			switch(op)
 				if("+")
-					result += val
+					result = (result + val)
```

Comment:
Or, how about, the next time somebody makes a pr where they make syntax changes for no fucking reason, you all fucking close it.